site/_posts/2019-10-01-implementing-fusionauth-python.md

```diff
@@ -529,22 +541,19 @@ If successful, you should see a response that looks something like this:
     ],
     "verified": true,
     "tenantId": "31626131-3938-6231-3634-663636656635",
-    "username": "terry@example.com",
-    "mobilePhone": "4245551212",
+    "username": "jerryexample",
     "passwordChangeRequired": false,
-    "fullName": "Terry Example",
+    "fullName": "Jerry Example",
     "insertInstant": 1569515252886,
     "lastName": "Example",
-    "twoFactorEnabled": false,
-    "email": "terry@example.com"
+    "email": "jerry@example.com"
   }
 }
 ```
 
-Navigate to **Users** in the FusionAuth UI. You should now see a new user created and registered to an application!
-
-{% include _image.liquid src="/assets/img/blogs/manage-user-screen.png" alt="Manage User Screen" class="img-fluid full mb-4" figure=false %}
+Navigate to "**Users**" in the FusionAuth UI. You should now see a new user created and registered to an application!
 
+{% include _image.liquid src="/assets/img/blogs/2021-add-user-api.png" alt="Manage User Screen" class="img-fluid full mb-4" figure=false %}
```

Comment:
All images for this should be in a subdirectory of blogs. Something like `img/blogs/fusionauth-python-app` or something similar.

Reply:
moved in latest commit